wwwapp/urls.py

```diff
@@ -28,9 +28,10 @@
     url(r'^article/(?P<name>[a-zA-Z0-9\-_]+)/$', views.article_view, name='article'),
     url(r'^articleNameList/$', views.article_name_list_view, name='articleNameList'),
     url(r'^addArticle/$', views.article_view, name='addArticle'),
-    url(r'^workshop/priv/(?P<name>[a-zA-Z0-9\-_]+)/$', views.workshop_view, name='workshop'),
     url(r'^workshop/(?P<name>[a-zA-Z0-9\-_]+)/$', views.workshop_page_view, name='workshop_page'),
-    url(r'^workshop/participants/(?P<name>[a-zA-Z0-9\-_]+)/$', views.workshop_participants_view, name='workshop_participants'),
+    url(r'^workshop/(?P<name>[a-zA-Z0-9\-_]+)/priv/$', views.workshop_view, name='workshop'),
+    url(r'^workshop/(?P<name>[a-zA-Z0-9\-_]+)/edit$', views.workshop_page_edit_view, name='workshop_page_edit'),
```

Comment:
Can we have any kind of consistency in these URLs please? All of them end with `/` except for these two...

Reply:
Consistency what is that? XD

Reply:
regularity, consistency, uniformity, conformity, symmetry, constancy, whichever one you prefer :P